pkg/apis/serving/v1alpha1/revision_types.go

```diff
@@ -178,7 +178,7 @@ type RevisionSpec struct {
 	// Revision. Defaults to Multi.
 	// Deprecated in favor of ContainerConcurrency.
 	// +optional
-	ConcurrencyModel RevisionRequestConcurrencyModelType `json:"concurrencyModel,omitempty"`
+	DeprecatedConcurrencyModel RevisionRequestConcurrencyModelType `json:"concurrencyModel,omitempty"`
```

Comment:
You'll need to update the godoc for the various things - ie. the doc here still uses `ConcurrencyModel`

Reply:
Done.